dokumentlager/domene/src/main/java/no/nav/foreldrepenger/melding/vilkår/Avslagsårsak.java

```diff
@@ -20,6 +20,7 @@
 
 @JsonFormat(shape = Shape.OBJECT)
 @JsonAutoDetect(getterVisibility = Visibility.NONE, setterVisibility = Visibility.NONE, fieldVisibility = Visibility.ANY)
+@SuppressWarnings("all")
```

Comment:
I stedet for "all" bør du bruke den konkrete i regelen som er angitt øverst til høyre i "Why is this an issue?"

Reply:
Ok